client/sections.js

```diff
@@ -289,12 +290,21 @@ const sections = [
 		secondary: true,
 	},
 	// this MUST be the first section for /read paths so subsequent sections under /read can override settings
+	{
+		name: 'entry',
+		paths: [ '/' ],
+		module: 'entry',
+		secondary: true,
+		group: 'reader',
```

Comment:
Something seems off with this empty section. It's a reader section that belongs to both the calypso and jetpack cloud apps. Seems like it's there to cover that something isn't setup properly at a framework level 😉